hnn_core/extracellular.py

```diff
@@ -403,7 +432,7 @@ def plot(self, *, trial_no=None, contact_no=None, tmin=None, tmax=None,
         fig : instance of plt.fig
             The matplotlib figure handle into which time series were plotted.
         """
-        from .viz import plot_extracellular
+        from .viz import plot_laminar_lfp
```

Comment:
Based on a discussion with @ntolley and @jasmainak, I've changed the name here to emphasize that this plotting function, along with its CSD counterpart, is specifically designed to plot the *laminar* profile of linear, equally spaced electrode contacts. I just now need to implement a check for the linear and equally spaced conditions.